rasa/nlu/model.py

```diff
@@ -93,14 +93,14 @@ def get(self, property_name: Text, default: Any = None) -> Any:
         return self.metadata.get(property_name, default)
 
     @property
-    def component_classes(self):
+    def component_classes(self) -> List[Optional[Any]]:
```

Comment:
we get a list of component class names, no?
```suggestion
    def component_classes(self) -> List[Optional[Text]]:
```